app/services/prison_visits/api.rb

```diff
@@ -58,6 +58,11 @@ def cancel_visit(id)
       response = @client.delete("visits/#{id}")
       Visit.new(response.fetch('visit'))
     end
+
+    def create_feedback(params)
+      @client.post('/feedback', params)
```

Comment:
How about passing a `FeedbackSubmission` into this method. Then, the knowledge of what parameters this API expects is contained entirely in this class, and `FeedbackSubmission` would need no knowledge of the API. The controller would simply call `api.create_feedback(feedback_submission)`.